overlord/ifacestate/ifacestate_test.go

```diff
@@ -5707,7 +5707,7 @@ func (s *interfaceManagerSuite) TestHotplugAddGoneSlot(c *C) {
 	slot := repo.Slot("core", "hotplugslot-old-name")
 	c.Assert(slot, NotNil)
 	c.Check(slot.Attrs, DeepEquals, map[string]interface{}{"foo": "bar"})
-	c.Check(slot.HotplugKey, Equals, "1234")
+	c.Check(slot.HotplugKey, DeepEquals, snap.HotplugKey("1234"))
```

Comment:
Why is this the case that just above this changed fragment we can use `Equals` and here the test was updated to `DeepEquals`?

Reply:
Not really needed, but see earlier explanation.